view/frontend/web/css/styles.css

```diff
@@ -100,3 +100,12 @@
     width: 225px;
     margin-bottom:18px;
 }
+
+#omise_offline_conveniencestorephoneNumber {
+    width: 225px;
+}
+
+#omise_offline_conveniencestoreemail {
+    width: 225px;
+    margin-bottom:18px;
```

Comment:
As Convenience Store payment and TrueMoney Wallet are sharing the similar stylesheet. Maybe we can make a generic class for this?